documentation/writing-providers.md

```diff
@@ -317,6 +368,7 @@ These are the things we'll be checking when you submit the PR.  Please try to co
 5. Re-run the [integration test](#step-7-integration-test) one last time.
   * Post the results as a comment to your PR.
 6. Re-read the [maintainer's responsibilities](providers.md#providers-with-contributor-support) bullet list.  By submitting a provider you agree to maintain it, respond to bugs, periodically re-run the integration test to verify nothing has broken, and if we don't hear from you for 2 months we may disable the provider.
+7. [Create an issue (feature request)](https://github.com/StackExchange/dnscontrol/issues/new?title=Add%20label%20for%20PROVIDERNAME) with the text "Please create the GitHub label 'provider-PROVIDERNAME'".
```

Comment:
**Idea**: Is it an idea to put this more as a checklist for us? Also in a review I feel the need to be able to tick off a checklist.

Reply:
It might make more sense to include the (create the GitHub label 'provider-PROVIDERNAME') request in the original GitHub pull request.